package.json

```diff
@@ -92,7 +92,7 @@
         "maplibre-gl": "^1.15.2",
         "matrix-encrypt-attachment": "^1.0.3",
         "matrix-events-sdk": "0.0.1",
-        "matrix-js-sdk": "github:matrix-org/matrix-js-sdk#develop",
+        "matrix-js-sdk": "github:matrix-org/matrix-js-sdk#cfe9bf8d4d59548c5f9bb904fc1da9ac79e58429",
```

Comment:
Temporarily set for deployment and tests to work

Reply:
Why not relying on branch matching?

Reply:
Do you mean set a branch here or is there something else that uses a specific JS SDK branch for the PR?